packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -804,6 +804,7 @@ namespace KB = KokkosBatched::Experimental;
           // CUDA_SAFE_CALL(cudaStreamAddCallback(stream_at_i, callback_isend, (void*)&data_at_i, 0));
         }
 
+        // FENCE REVIEW: Not Tested (in #ifdef IFPACK2_BLOCKTRIDICONTAINER_USE_CUDA_STREAM)
```

Comment:
@kyungjoo-kim Can you comment on this fence?

Reply:
The stream version pack/unpack is not activated in the code. So, fence or no fence does not change the code behavior. I am actually working on this part and will handle this myself.